vms/avm/state/state.go

```diff
@@ -150,9 +127,8 @@ type state struct {
 	utxoDB        database.Database
 	utxoState     avax.UTXOState
 
-	statusesPruned bool
-	statusCache    cache.Cacher[ids.ID, *choices.Status] // cache of id -> choices.Status. If the entry is nil, it is not in the database
-	statusDB       database.Database
+	statusCache cache.Cacher[ids.ID, *choices.Status] // cache of id -> choices.Status. If the entry is nil, it is not in the database
+	statusDB    database.Database
```

Comment:
I think we should be able to remove this entirely right?

Reply:
Removed